app/PimRequirements.php

```diff
@@ -64,101 +64,6 @@ public function getPimRequirements(): array
     }
 
     /**
-     * Gets the MySQL server version thanks to a PDO connection.
```

Comment:
it's performed directly in the `CommunityRequirements` class (located in CE dev)